java/code/src/com/redhat/rhn/frontend/strings/jsp/StringResource_bn_IN.xml

```diff
@@ -16262,7 +16262,7 @@ but are not subscribed to _@@PRODUCT_NAME@@Config_.</context>
         </context-group>
       </trans-unit>
       <trans-unit id="deployconfirm.jsp.h2" xml:space="preserve">
-        <source>Deploy Configuration Files: Comfirm Fields and Systems for Deployment</source>
+        <source>Deploy Configuration Files: Confirm Fields and Systems for Deployment</source>
```

Comment:
please change only StringResource_en_US.xml
The others will do it when we do the next translation update

Reply:
Thanks for the heads-up.
Rebased